StRoot/StFttDbMaker/StFttDb.h

```diff
@@ -0,0 +1,133 @@
+/***************************************************************************
+ * $Id: StFttDb.h,v 1.3 2021/05/27 14:02:23 akio Exp $
+ * \author: akio ogawa
+ ***************************************************************************
+ *
+ * Description: FCS DB Utility
```

Comment:
FCS?

Reply:
Addressed in 3686a6eea3b300c95180bdcee0ad698b5763a88c